projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/qualifying-questions/qualifying-question-options/qualifying-question-options.component.html

```diff
@@ -0,0 +1,30 @@
+<ng-container *ngIf="qualifyingQuestionsControl">
+  <div class="govuk-form-group" [class.govuk-form-group--error]="displayError">
+    <fieldset class="govuk-fieldset">
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+        <h1 class="govuk-fieldset__heading">
+          {{ 'Are you contacting us about any of the following:' | rpxTranslate }}
```

Comment:
@johnbenjamin-hmcts - quick question... how come this appears here and in [qualifying-question-detail.component.html](https://github.com/hmcts/ccd-case-ui-toolkit/pull/1500/files#diff-79c2f173b055c7b911b504225c405ffeafc82a4f63bcc2a89b2ff5152452acd1)?

Reply:
@Daniel-Lam - In the radio options selection component, this will be the main heading, and in the markdown display component, this will become a greyed-out heading on top of the main heading.

Reply:
Ah, OK - makes sense, @johnbenjamin-hmcts... thanks